stl/CMakeLists.txt

```diff
@@ -386,6 +386,10 @@ set(SOURCES_SATELLITE_2
     ${CMAKE_CURRENT_LIST_DIR}/src/special_math.cpp
 )
 
+set(SOURCES_SATELLITE_3

```

Comment:
Done. Hopefully, it will be renamed to something like `_CXX20_SYCNH`, there's `<semaphore>`, `<latch>`, and `<barrier>` and they look like easier than atomic_wait, if they can use atomic_wait.